java-manta-client-kryo-serialization/pom.xml

```diff
@@ -120,6 +86,24 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.objenesis</groupId>
+            <artifactId>objenesis</artifactId>
+            <version>${dependency.objenesis.version}</version>
+            <scope>compile</scope>
+        </dependency>
+
+        <!-- Fast-md5 is added in the provided scope because it is shaded and
+             not relocated. We don't want it to be transitively passed on as a
+             dependency, so it is used in the runtime scope.
+        -->
+        <dependency>
+            <groupId>com.twmacinta</groupId>
```

Comment:
Change `<groupId>com.twmacinta</groupId>` to `<groupId>com.joyent.util</groupId>` here.

Reply:
👍 Please commit that change.

Reply:
Also, PR #285 hasn't effectively solved the issue and it requires more work.

Reply:
A new branch `fixes-270` has been created in my forked [repository](https://github.com/nairashwin952013/java-manta) for proactively tracking this issue.